tools/fetch_elfutils.sh

```diff
@@ -51,14 +55,16 @@ mkdir src
 cd src
 tar --no-same-owner --strip-components 1 -xf "../${TAR_ELF}"
 
-echo "Compiling elfutils using ${C_COMPILER}"
-
 # The flags below are hardcoded to work around clang compatibility issues in
 # elfutils 186; these are irrelevant for GCC.  Note that this won't propagate
 # envvars back up to the calling build system.
 # TODO is there a better way to infer compiler family?
 if [[ "$(basename "${C_COMPILER}")" == clang* ]]; then
   export CFLAGS="-Wno-xor-used-as-pow -Wno-gnu-variable-sized-type-not-at-end -Wno-unused-but-set-parameter"
 fi
+export CFLAGS="${CFLAGS-""} ${EXTRA_CFLAGS}"
```

Comment:
I think this is good, although we don't do anything here to reconcile conflicting options.  I think that's good because dealing with conflicts is hard, just pointing it out.